webapp/containers/add_users_to_team.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {getProfilesNotInTeam} from 'mattermost-redux/actions/users';
+
+import AddUsersToTeam from 'components/add_users_to_team.jsx';
+
+function makeMapStateToProps() {
```

Comment:
I just copied what the RN files were doing such as https://github.com/mattermost/mattermost-mobile/blob/master/app/components/post/index.js#L19 but I guess I copied the wrong one, since that needs it for the factory selector. I'll switch them over